### PR TITLE
Setup Gtk nuget packages

### PR DESCRIPTION
**Pull request by cra0zy**

### CODE REVIEW

**Comment by cwensley** [comment]
Thanks for updating the nuget packages!

I don't think it's building the new Gtk2 nupkg.  You'll need to add an entry [here](https://github.com/picoe/Eto/blob/develop/build/Build.proj#L132-L134) to do that.

File: build/nuspec/Eto.Platform.Gtk.nuspec [hunk 1]
```diff
@@ -24,11 +24,12 @@ On Linux, mono framework 2.10 or higher and gtk-sharp2 are required.
 		<dependencies>
 			<group>
 				<dependency id="Eto.Forms" version="[$version$]" />
+				<dependency id="GtkSharp" version="3.22.24.28" />
 			</group>
 		</dependencies>
 	</metadata>
 	<files>
-		<file src="artifacts\core\$configuration$\net45\Eto.Gtk2.dll" target="lib\net45" />
+		<file src="artifacts\core\$configuration$\netstandard2.0\Eto.Gtk.dll" target="lib\net45" />
```
Comment (cwensley) [inline build/nuspec/Eto.Platform.Gtk.nuspec]: the target folder here should probably be `lib\netstandard2.0`

**Comment by cra0zy** [comment]
> I don't think it's building the new Gtk2 nupkg. You'll need to add an entry here to do that.

Already connected.

**Comment by cwensley** [comment]
beautiful!

**Pull request merged**